code/API_definitions/location-subscriptions.yaml

```diff
@@ -0,0 +1,591 @@
+      operationId: getEventSubscription
+      description: Retrieve a given event subscription by ID
+      parameters:
+        - name: eventSubscriptionsId
+          in: path
+          description: event subscription identifier that was obtained from the create event subscription operation
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Contains information about Event Subscriptions
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SubscriptionInfo'
+        '400':
+          $ref: '#/components/responses/Generic400'
+        '401':
+          $ref: '#/components/responses/Generic401'
+        '403':
+          $ref: '#/components/responses/Generic403'
+        '404':
+          $ref: '#/components/responses/Generic404'
+        '500':
+          $ref: '#/components/responses/Generic500'
+        '503':
+          $ref: '#/components/responses/Generic503' 
+    delete:
+      tags:
+        - Location Event Subscription
+      summary: 'Operation to delete a event subscription'
+      operationId: deleteEventSubscription
+      description: Delete a given subscription of event by ID
+      parameters:
+        - name: eventSubscriptionsId
+          in: path
+          description: event subscription identifier
+          required: true
+          schema:
+            type: string
+      responses:
+        '204':
+          description: Event subscription deleted
+        '400':
+          $ref: '#/components/responses/Generic400'
+        '401':
+          $ref: '#/components/responses/Generic401'
+        '403':
+          $ref: '#/components/responses/Generic403'
+        '404':
+          $ref: '#/components/responses/Generic404'
+        '500':
+          $ref: '#/components/responses/Generic500'
+        '503':
+          $ref: '#/components/responses/Generic503'
+ 
+components:
+  securitySchemes:
+    oAuth2ClientCredentials:
+      type: oauth2
+      flows:
+        clientCredentials:
+          tokenUrl: '{tokenUrl}'
+          scopes: {}
+    notificationsBearerAuth:
+      type: http
+      scheme: bearer
+      bearerFormat: "{$request.body#/webhook/notificationAuthToken}"      
+    three_legged:
+      type: oauth2
+      flows:
+        authorizationCode:
+          authorizationUrl: https://auth.example.com/authorize
+          tokenUrl: https://auth.example.com/token
+          scopes: {}
+  
+  schemas:
+    Area:
+      oneOf: 
+        -  $ref: "#/components/schemas/Circle"
+      discriminator: 
+        propertyName: type
+        mapping: 
+          Circle:  "#/components/schemas/Circle"
+    
+    Circle:
+      type: object
+      description: Circular area
+      required:
+        - type
+        - location
+        - accuracy
+      properties:
+        type:
+          type: string
+          description:  Type of this area.
+        location:
```

Comment:
For the 2 other API we changed to  `center` to make ot more explicit

Reply:
I changed the location property inside Circle to center